### ISSUE TITLE
Releases?

### ISSUE THREAD

**Issue by exadeci** [peregin/gps-overlay-on-video#39]
This looks like a really cool tool to easily add custom huds on gopro videos but the last release was published 4 years ago.

Would it be possible to have one without having to set up a scala development environment on our computers?

https://github.com/scalacenter/sbt-release-early/wiki/How-to-release-in-Travis-(CI) could help.

**Comment by peregin** [comment]
Hi @exadeci ,
Thanks for the feedback and interest. As a very quick solution I've added the sbt release plugin https://github.com/sbt/sbt-release and triggered a fresh release v1.1.0
I leave the ticket open and will setup travis ci.
Have a great weekend!
Best regards,
Levente

**Comment by exadeci** [comment]
@peregin Thank you for the super quick reply!
I had not paid attention to the fact that the first release was just the source code, what I meant was if you could release an executable/.jar so people don't need to setup sbt and the scala environment.

Thanks

**Comment by peregin** [comment]
hi @exadeci ,
I've updated the build.sbt, with the combination of sbt-release, sbt-assembly and sbt-github-release it is possible to trigger a release and the artifacts are uploaded to the new Github release.
Also I've added a latest release badge to the readme markdown, pointing to the latest released entry.
[![GitHub release](https://img.shields.io/github/release/peregin/gps-overlay-on-video.svg)](https://github.com/peregin/gps-overlay-on-video/releases)
The fat/uber jar is uploaded, it can be easily started without a scala environment.
Have a nice evening!
Levente
